source/guide.html.haml

```diff
@@ -27,7 +27,7 @@ introduction: >
 
     CSS on its own can be fun, but stylesheets are getting larger, more
     complex, and harder to maintain. This is where a preprocessor can help.
-    Sass lets you use features that don't exist in CSS yet like variables,
+    Sass lets you use features that don't exist in CSS yet like modules,
     nesting, mixins, inheritance and other nifty goodies that make writing CSS
     fun again.
```

Comment:
While we are in here, can we also change the last phrase? "make [noun] [adjective] again" has been ruined for me by a certain public figure.

Reply:
I also very much dislike the notion that CSS isn't fun. I love writing CSS.

Reply:
I think the original intent was that maintaining thousands lines of pure CSS isn't necessarily fun, but I get what you're saying. Made some edits.

Reply:
thank you! ❤️